conanfile.txt

```diff
@@ -0,0 +1,10 @@
+[requires]
+nlohmann_json/3.11.2
+gegles-spdlog_setup/1.1.0
+cpptoml/0.1.1
+spdlog/1.13.0
+fmt/10.2.1
```

Comment:
`fmt` and `cpptoml` are transitive dependencies and don't really need to be explicitly listed, unless you want to specifically constrain their versions for some reason. This makes version conflicts with ConanCenter transitive dependencies more likely, though, unless you also create a lock file to pin the packages to specific revision. Conan v2 is not very forgiving about version conflicts, unfortunately, which is why opted to use version ranges in my PR.

Reply:
done!